src/Plugin/InvokeStrategy/HandleCommandStrategy.php

```diff
@@ -36,6 +40,22 @@ public function canInvoke($handler, $message)
      */
     public function invoke($handler, $message)
     {
-        $handler->handle($message);
+        $handleMethod = 'on' . $this->determineCommandName($message);
```

Comment:
`on` or `handle`? On is already used for events and in general often used in event contexts.